Jenkinsfile

```diff
@@ -43,6 +40,11 @@ node('f28-os') {
         sh 'cat GIT_COMMIT'
         def commitHash = readFile('GIT_COMMIT').trim()
 
+        sh "sed s/BUILD_VERSION_PLACEHOLDER/${BUILD_VERSION}/ release_info.json > temp_release_info.json"
```

Comment:
There may be limitations because of Jenkins. 

However, you can use `sed -i` to find & replace within the file. 
`sh "sed -i s/BUILD_VERSION_PLACEHOLDER/${BUILD_VERSION}/g release_info.json"`

For example, I tried this locally: 
```
[root@rhel8 django-rest-framework]# sed -i s/BUILD_VERSION_PLACEHOLDER/0.0.9/g release_info.json
[root@rhel8 django-rest-framework]# cat release_info.json
{
	"version": "0.0.9"
}
[root@rhel8 django-rest-framework]#
```
That way you don't have to do a `sh "mv temp_release_info.json release_info.json"` right after.

Reply:
removed that